src/components/PersonLink.tsx

```diff
@@ -0,0 +1,25 @@
+import { NavLink } from 'react-router-dom';
+import { Person } from '../types';
+
+interface Props {
+  person?: Person;
+}
+
+const PersonLink: React.FC<Props> = ({ person }) => {
+  if (!person) {
+    return <span>-</span>;
+  }
+
+  const isFemale = person.sex === 'f';
+
+  return (
+    <NavLink
+      to={`/people/${person.slug}`}
+      className={`has-text-weight-bold ${isFemale ? 'has-text-danger' : ''}`}
```

Comment:
It's better to use classnames library (or smth similar) for such conditional classes